snippets/csharp/GettingStarted.cs

```diff
@@ -76,4 +76,27 @@ public void FetchNodeInfo(BlockingBreezServices sdk)
         }
         // ANCHOR_END: fetch-balance
     }
+
+    // ANCHOR: logging
+    public void GettingStartedLogging()
+    {
+        try
+        {
+            sdk = BreezSdkMethods.SetLogStream(new SdkLogStream());
+        }
+        catch (Exception)
+        {
+            // Handle error
+        }
+    }
+
+    class SdkLogStream : LogStream
+    {
+        public void Log(LogEntry l)
+        {
+            Console.WriteLine($"Received Log [{l.Level}]: {l.Line}");
```

Comment:
Re: CI errors
```
'LogEntry' does not contain a definition for 'Level'...
'LogEntry' does not contain a definition for 'Line'...
```

Is this supposed to be:
```suggestion
            Console.WriteLine($"Received Log [{l.level}]: {l.line}");
```